src/shared/lib/resources/cwu-opportunity.ts

```diff
@@ -0,0 +1,41 @@
+import { Addendum } from 'shared/lib/resources/addendum';
```

Comment:
@sutherlanda Can you rename this file to `.../resources/opportunity/code-with-us.ts` to match the naming convention in the front-end?